src/js/msp/MSPHelper.js

```diff
@@ -147,6 +147,13 @@ MspHelper.prototype.process_data = function(dataHandler) {
                     MOTOR_DATA[i] = data.readU16();
                 }
                 break;
+            case MSPCodes.MSP_MOTOR_OUTPUT_REORDERING:
+                MOTOR_OUTPUT_ORDER = [];
```

Comment:
Use `window.MOTOR_OUTPUT_ORDER` to make clear that is global variable defined outside. I think it will work. The ideal is that FC will be a global object so here we can use `FC.MOTOR_OUTPUT_ORDER` but this change is outside this PR.